### PR TITLE
Document disabling default import rule

### PR DESCRIPTION
**Pull request by Tener**

### CODE REVIEW

**Review by greedy52 — APPROVED** [review]
should we update this reference too?

https://goteleport.com/docs/reference/resources/#database-object-import-rule

File: docs/pages/enroll-resources/database-access/rbac.mdx [hunk 1]
```diff
@@ -264,6 +264,35 @@ spec:
 version: v1
 ```
 
+### Disabling the default import rule
+
+Teleport expects at least one import rule to be defined. If it is missing, the auth server will create a default import rule on startup.
```
Comment (ptgott) [inline docs/pages/enroll-resources/database-access/rbac.mdx:269]:
```suggestion
Teleport expects at least one import rule to be defined. If it is missing, the Teleport Auth Service will create a default import rule on startup.
```
For consistent naming across the docs

File: docs/pages/enroll-resources/database-access/rbac.mdx [hunk 2]
```diff
@@ -264,6 +264,35 @@ spec:
...
+
+Create the custom rule and remove the default one:
+
+{/* spell-checker: disable */}
+```text
+> tctl create -f import_no_objects.yaml
+rule "import_no_objects" has been created
+> tctl rm db_object_import_rule/import_all_objects
+Rule "import_all_objects" has been deleted
+```
```
Comment (ptgott) [inline docs/pages/enroll-resources/database-access/rbac.mdx:293]:
````suggestion
```code
$ tctl create -f import_no_objects.yaml
rule "import_no_objects" has been created
$ tctl rm db_object_import_rule/import_all_objects
Rule "import_all_objects" has been deleted
```
````
To use our custom code snippet syntax

**Review by ptgott — APPROVED** [review]

**Comment by Tener** [comment]
> should we update this reference too?
> 
> https://goteleport.com/docs/reference/resources/#database-object-import-rule

I gave it some thought and I think we are better off _not_ doing that, as there is a risk of confusing readers.

**Comment by Tener** [comment]
@zmb3 This needs another approval, can you do the honors?

**Review by zmb3 — APPROVED** [review]

**Pull request merged**